projects/kit/abstract/input-slider.ts

```diff
@@ -143,6 +144,10 @@ export abstract class AbstractTuiInputSlider<T>
         this.updateHovered(hovered);
     }
 
+    private get decimalLimit(): number {
```

Comment:
```ts
private get decimalLimit(): number {
   const [, fractionPart = ''] = this.quantum.toString().split('.');
   return fractionPart.length || 2;
}
```

`2` - Do we need to move/provide this value to a constant?

Reply:
@splincode  thanks, done 👍  

@waterplea  getter is only called from TuiMapper depending on quantum. So I think another tuiPure would be redundant. What do you think?

Reply:
Right, it's already memoized. Let's change it to a function instead of getter, that gets `quantum` as an argument to keep mapping function pure.